lib/Conversion/TritonGPUToLLVM/Utility.cpp

```diff
@@ -465,7 +465,7 @@ largestVectorisation(MLIRContext *ctx, const LinearLayout &cvt, int bitwidth,
     }
     return {v, permutation};
   }
-  llvm_unreachable("No vectorisation found");
+  llvm_unreachable("Vectorization < 1 is not valid");
```

Comment:
This is probably clearer than "no vectorization" because "no vectorization" may include vec=1.

@lezcano , just my personal opinion, happy to revert back.